lib/src/fitness.dart

```diff
@@ -77,9 +77,9 @@ Future<Fitness> calcFitness(
 
   if (fileAnalyzerItems != null) {
     for (var item in fileAnalyzerItems) {
-      if (item.type.startsWith('INFO')) {
+      if (item.severity.startsWith('INFO')) {
         penalties += hintPoints;
-      } else if (item.type.startsWith('WARN')) {
+      } else if (item.severity.startsWith('WARN')) {
```

Comment:
Ditto - can now be `==` now

Reply:
is it 'WARN' or 'WARNING'? I wasn't sure, hence the startsWith